static/style.css

```diff
@@ -53,3 +53,24 @@ footer.ui.footer {
 .size {
     font-size: 15px;
 }
+
+.ui.input.focus input,
+.ui.input input:focus {
+  border-color: #85B7D9;
+  background: #FFFFFF;
+  color: rgba(0, 0, 0, 0.8);
+  box-shadow: none;
+}
+.ui.input.focus input::-webkit-input-placeholder,
+.ui.input input:focus::-webkit-input-placeholder {
+  color: rgba(128, 128, 128, 1);
+}
+.ui.input.focus input::-moz-placeholder,
```

Comment:
but I thought you can't group selectors if one if them is invalid?

On Wed, Feb 10, 2016 at 6:25 PM, Jin Park <notifications@github.com> wrote:

> In static/style.css
> <https://github.com/billyfung/tellmeabout.coffee/pull/79#discussion_r52558272>
> :
>
> > @@ -53,3 +53,24 @@ footer.ui.footer {
> >  .size {
> >      font-size: 15px;
> >  }
> > +
> > +.ui.input.focus input,
> > +.ui.input input:focus {
> > +  border-color: #85B7D9;
> > +  background: #FFFFFF;
> > +  color: rgba(0, 0, 0, 0.8);
> > +  box-shadow: none;
> > +}
> > +.ui.input.focus input::-webkit-input-placeholder,
> > +.ui.input input:focus::-webkit-input-placeholder {
> > +  color: rgba(128, 128, 128, 1);
> > +}
> > +.ui.input.focus input::-moz-placeholder,
>
> http://www.w3schools.com/css/css_syntax.asp
>
> look at grouping selectors
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/billyfung/tellmeabout.coffee/pull/79/files#r52558272>.
>